polartools/load_data.py

```diff
@@ -57,6 +92,7 @@ def load_databroker(scan_id, db, version='v2', **kwargs):
     data : pandas.DataFrame
         Table with the data from the primary stream.
     """
+
     # TODO: Not sure this is the best way to handle this.
     if 'databroker' not in str(type(db)):
```

Comment:
More natural to just `try` the code you want, then handle the fallback in the expected `except EXPECTED_EXCEPTION_NAME` segment.  Or, instead of comparing with `str(type(db))`, you might `isinstance(obj, databroker)`

Reply:
The reason I did this way (and added the TODO note) is that the v1 and v2 databrokers return different `type` (see below). Is there another way to check that an object is a databroker database?

```
In [13]: db = catalog['test_data']

In [14]: type(db)
Out[14]: databroker._drivers.msgpack.BlueskyMsgpackCatalog

In [15]: db2 = Broker.named('test_data')

In [16]: type(db2)
Out[16]: databroker.v1.Broker
```

Reply:
When you want a `databroker.Broker`, you can check `isinstance(db, databroker.Broker)` and similar `isinstance(db, databroker.catalog)`.  But why bother with the checks?  Each of these objects has provided a converter to get to the other API so you do not have to care which `db` version you have.

To get a Broker (old style), `db.v1`.
To get a catalog (new style), `db.v2`.

Reply:
My idea here was to just catch the case where db is not a database, but looking again at it, you are right that this check is unnecessary in this function.